services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -255,18 +230,7 @@ export const AddEditReportModal = ({
           >
             Continue from previous period
           </Button>
-          {isCopyDisabled() ? (
-            <Button
-              sx={sx.close}
-              onClick={writeReport}
-              disabled={submitting}
-              type="submit"
-              variant="outline"
-              data-testid="modal-logout-button"
-            >
-              Start new
-            </Button>
-          ) : (
+          {!isCopyDisabled() && (
```

Comment:
let me know if this logic was supposed to change! i just saw we removed the first option but kept the second, so i flipped the boolean to match